### PR TITLE
feat(routing): start generating apiv2

### PR DESCRIPTION
**Pull request by quartzmo**

### CODE REVIEW

File: internal/gapicgen/generator/config.go [hunk 1]
```diff
@@ -280,8 +280,7 @@ var MicrogenGapicConfigs = []*MicrogenConfig{
 		ApiServiceConfigPath:  "bigqueryreservation_v1beta1.yaml",
 		Transports:            []string{"grpc", "rest"},
 		ReleaseLevel:          "beta",
-	},
-	{
+	}, {
```
Comment (codyoss) [inline internal/gapicgen/generator/config.go:283]: nit: this looks unintentional

File: routing/go.mod [hunk 2]
```diff
@@ -0,0 +1,26 @@
+module cloud.google.com/go/routing
```
Comment (codyoss) [inline routing/go.mod:1]: I bet we will eventually be generating more maps APIs. I think we should make the module under `cloud.google.com/go/maps` and have routing be a subdir

**Review by codyoss — CHANGES_REQUESTED** [review]

File: routing/go.mod (hunk 2 above)
Comment (quartzmo) [reply routing/go.mod:1]: Great, will do.

Comment (quartzmo) [reply routing/go.mod]: Fixed, thanks!

File: internal/gapicgen/generator/config.go (hunk 1 above)
Comment (quartzmo) [reply internal/gapicgen/generator/config.go]: Fixed, thanks!

**Review by codyoss — APPROVED** [review]

**Pull request merged**